edx-platform/pearson-vue-theme/lms/templates/header/navbar-authenticated.html

```diff
@@ -14,6 +14,7 @@
   show_explore_courses = settings.FEATURES.get('COURSES_ARE_BROWSABLE')
   self.real_user = getattr(user, 'real_user', user)
   enable_help_link = settings.FEATURES.get('ENABLE_HELP_LINK')
+  show_programs_tab = configuration_helpers.get_value('SHOW_PROGRAMS_TAB', show_program_listing)
```

Comment:
Are we using this setting in any of the prod sites? @sergivalero20

Reply:
Yes:

![image](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/3fdb4ccf-ef7b-4525-a8be-88fa6886a8cf)